src/main/java/com/antonjohansson/versionlifecycleplugin/ValidateMojo.java

```diff
@@ -0,0 +1,24 @@
+package com.antonjohansson.versionlifecycleplugin;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+/**
+ * Validates the project. Binds to the {@code version-validate} phase.
+ */
+@Mojo(name = "validate")
+public class ValidateMojo extends AbstractVersionMojo
+{
+    @Parameter(defaultValue = "${checkForUncommittedChanges}")
```

Comment:
Here's how we've done for other parameters:
```java
@Parameter(name = "releaseCommitMessagePattern", property = "version.releaseCommitMessagePattern", defaultValue = "[version]")
```

So I suggest:

```diff
-    @Parameter(defaultValue = "${checkForUncommittedChanges}")
+    @Parameter(name = "checkForUncommittedChanges", property = "version.checkForUncommittedChanges", defaultValue = "true")
```

Reply:
Changed!